### DISCUSSION TITLE
About the expectedOrigin

### DISCUSSION THREAD

**Discussion by laleetborse** [MasterKale/SimpleWebAuthn#474] [Troubleshooting]
Hello guys I used this library for my project and the backend is build separately but now I am facing the problem that it is not able to work as the deployed frontend does not have the same "expectedOrigin" how to get rid of this problem, I tried to use the request body to make the allow any origin but it is not a good solution can any one help??

**Comment by spendres** [comment]
You can populate the RPID using the url of the host read through the runtime. This is how many have supported running integration tests on CI/CD environments whose hostname changes each time the library runs.

**Reply by laleetborse** [reply]
any fix technique that I can use 
why it is soo important to use this expectedOrigin  the cors can also handle the security issue.
